OMGVA PoS/Data layer/Mappers/TaxMapper.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OMGVA_PoS.Data_layer.DTOs;
+using OMGVA_PoS.Data_layer.Models;
+
+namespace OMGVA_PoS.Data_layer.Mappers;
+
+public class TaxMapper
```

Comment:
This whole class looks suspicious. All these methods will probably used only once. Also, they couuld be replaced by casting, maybe?

Reply:
Although it _might_ be that all of the methods are used only once, this is for separation of concerns. Why would we add mapping logic to Controller class, the code would become a tangled mess. Although for taxes it's just a couple fields, other models will require bigger methods. Furthermore, if we will want to add TaxItems collection in the response, the mapping logic would become more difficult

Reply:
It still seems unnecessarily complex for no reason. But fine, it can stay for now.